Source/Configuration/YPFonts.swift

```diff
@@ -33,7 +33,7 @@ public struct YPFonts {
 
     public var cameraTimeElapsedFont: UIFont = .monospacedDigitSystemFont(ofSize: 13, weight: .medium)
 
-    public var navigationBarTitleFont: UIFont?
+    public var navigationBarTitleFont: UIFont = .boldSystemFont(ofSize: 17)
```

Comment:
I don't think the folt should be bold by default

Reply:
What is your suggestion? I'm using this because I'm trying to adapt to the design of the library, I saw they're using this font type and size in the picker title font.

```
/// The font used in the picker title
    public var pickerTitleFont: UIFont = .boldSystemFont(ofSize: 17)
```

Reply:
Oh, you are right! I did not pay attention to this detail, I'm sorry haha.